src/app/demo.component.ts

```
@@ -0,0 +1,58 @@
+@Component({
+  selector: "app-greeting", // How you'll use this component in other templates: <app-greeting></app-greeting>
+  templateUrl: "./greeting.component.html", // Links to the HTML template
+  styleUrls: ["./greeting.component.css"], // Links to component-specific CSS (optional)
+})
+export class GreetingComponent implements OnInit {
+  // Properties (data) that can be displayed or manipulated
+  title: string = "Welcome to My Angular App!";
+  userName: string = "Guest";
+  initialUserName: string = "Guest"; // To store the initial state for reset
+  showDetails: boolean = false;
+  showDetails: boolean = false;
```

Biome (2.1.2)

[error] Duplicate class member name "showDetails"



(lint/suspicious/noDuplicateClassMembers)